### DISCUSSION TITLE
Quickly access mono audio toggle in eartrumpet itself

### DISCUSSION THREAD

**Discussion by N1N74** [File-New-Project/EarTrumpet#939] [Ideas]
Toggling mono quickly and seeing whether mono audio is turned on from eartrumpet would be great. I'm having to toggle it from settings everytime I need to.

**Comment by RedSnt** [comment]
Not a lot of love for this one, but I'd personally enjoy this setting. Every now and then, at least once a month, I come across videos with poor sound mixing, where all the sound is only on one ear. Would be nice with a quick toggle in eartrumpet, doesn't even have to be per application in my case.
